src/libs/utils.ts

```diff
@@ -78,3 +78,15 @@ export const isValidNetworkAddress = (maybeAddress: string): boolean => {
   }
   return false
 }
+
+export const formatBytes = (bytes: number, decimals = 2): string => {
+  if (!+bytes) return '0 Bytes'
```

Comment:
is there a reason for this + here ?

Reply:
It's the unary operator. It's used to attempt to convert the variable to a number. As we are using typescript here, it doesn't make much sense. Will remove before merging.